contracts/raffle/FlowtyRaffles.cdc

```diff
@@ -34,62 +34,66 @@ pub contract FlowtyRaffles {
     there is not way for the raffle itself to know if a source is acting in good faith. Make sure you choose a raffle
     source implementation with care, as choosing one from an unknown party could result in unfair outcomes.
     */
-    pub resource interface RaffleSourcePublic {
+    access(all) resource interface RaffleSourcePublic {
         /*
         getEntries - return all entries in this raffle.
         NOTE: This will not work if a raffle is so large that returning all its entries will exceed computation limits
         */
-        pub fun getEntries(): [AnyStruct]
+        access(all) fun getEntries(): [AnyStruct]
 
         // getEntryCount - return the total number of entries in this raffle source
-        pub fun getEntryCount(): Int
+        access(all) fun getEntryCount(): Int
 
         // getEntryAt - return the entry at a specific index of a raffle source
-        pub fun getEntryAt(index: Int): AnyStruct
+        access(all) fun getEntryAt(index: Int): AnyStruct
     }
 
-    pub resource interface RaffleSourcePrivate {
+    access(all) entitlement RevealRaffle
+    
+    access(all) entitlement AddEntry
```

Comment:
Let's move these two further up in the file itself, most of the time I see these right after event definitions

We should also add some comments about each entitlement and what it's for